src/api.rs

```diff
@@ -309,13 +326,15 @@ pub enum ChromaSamplePosition {
   Colocated
 }
 
+/// Assign default ChromaSamplePosition as Unknown
 impl Default for ChromaSamplePosition {
   fn default() -> Self {
     ChromaSamplePosition::Unknown
   }
 }
 
 arg_enum!{
+  /// Contains details of Pixel color range
```

Comment:
It does not add more information, you may add a reference to what is a color range is and the ITU H.273 section that mentions it.